openbb_terminal/README.md

```diff
@@ -186,41 +188,18 @@ Several features in this project utilize Machine Learning. Machine Learning Pyth
 
    To enable the `prediction` menu install additional dependencies after installing main dependencies:
 
-   - On Intel mac
+   - On Intel and M1 macs
 
      ```bash
-     conda install -c conda-forge tensorflow==2.7.0
-     conda install -c conda-forge -c pytorch u8darts-torch
+     conda install -c conda-forge tensorflow==2.7.1
+     conda install -c pytorch u8darts-torch
      poetry install -E prediction
      ```
 
-   - On M1 mac (make sure you are using miniconda3 or miniforge3)
```

Comment:
These were the commands to get tensorflow + torch properly installed on M1. Is there a reason why this is being removed?

Reply:
We don't support Apple Silicon as an independent platform at the moment.

Getting u8darts-torch and torch from a conda channel where the binaries are supported by both Apple Silicon and Intel architectures should work for all mac users.

We should be looking at adding support for Apple Silicon when it works out of the box without the need to build from source.